packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
We can try that, but I'm not sure it's possible.  I think the way gradle works is that it uses shells scripts to bootstrap a java build of a tool that then builds your code.  This is code that gets linked into the Java tool, and we don't want to mess with their shells scripts, so there might not be a way to jam it in later.  Maybe it's possible to reference with some sort of path?  It seemed to be using Java-style magic directories when I looked at it before.